scripts/datafiles/build_datafile_ftp_directory.pl

```diff
@@ -222,16 +277,18 @@ sub _target_species_root {
   return File::Spec->catdir($base, $file_type, $species, $ftp_type);
 }
 
+
 # Expected path: base/data_files/normalpath
 # e.g. pub/release-66/data_files/pan_trogladytes/CHIMP2.14/rnaseq/chimp_1.bam
 sub _target_datafiles_root {
   my ($self, $datafile) = @_;
   my $base = File::Spec->catdir($self->opts()->{ftp_dir}, 'data_files');
-  my $target_location = $datafile->path($base);
+  (my $target_location = $datafile->path($base)) =~ s/funcgen\///;  
```

Comment:
I would prefer this to be two lines, as legibility is poor, and I can't guarantee Perl's behaviour. The brackets could mean list context as much as it controls operator precedence. I'm sure it works, but it makes me cautious.